sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -1038,10 +1100,13 @@ def process_timer(self, window_id, unused_name, time_domain, timestamp,
     window = state.get_window(window_id)
     if state.get_state(window, self.TOMBSTONE):
       return
-    if time_domain == TimeDomain.WATERMARK:
+
+    if (time_domain == TimeDomain.WATERMARK or
+        time_domain == TimeDomain.REAL_TIME):
```

Comment:
<!--new_thread; commit:a68ceca8046b6d42f941aa4016a528ec2ee3808a; resolved:0-->
`time_domain in (TimeDomain.WATERMARK, TimeDomain.REAL_TIME)`.

Reply:
done